dynamic-graph-service/conf/bench/icbu-dataset/coordinator.yml

```diff
@@ -0,0 +1,33 @@
+schema-file: "../../conf/bench/icbu-dataset/schema.json"
+
+data-loading:
+  worker-num: 1
+
+sampling:
+  worker-num: 1
+  actor-local-shard-num: 36
+  store-partitions: 36
+  store-partition-strategy: "hash"
+  downstream-partition-strategy: "hash"
+
+serving:
+  worker-num: 1
+  actor-local-shard-num: 36
+  store-partitions: 36
+  store-partition-strategy: "hash"
+
+kafka:
+  dl2spl:
+    servers:
+      - "100.82.16.15:9092"
+      - "100.82.16.18:9092"
+      - "100.82.16.19:9092"
+    topic: "record-batches-e2e"
+    partitions: 12
+  spl2srv:
+    servers:
+      - "100.82.16.15:9092"
```

Comment:
Maybe we should think carefully about how to expose these ips.

Reply:
change to localhost